### PR TITLE
Fix the `NO_HTTP3_SKIP_TARGETS` variable

### PR DESCRIPTION
**Pull request by jwendell**
Replaces a duplicated item with the correct one.

### CODE REVIEW

**Comment by jwendell** [comment]
cc @danzh2010

File: bazel/repositories.bzl [hunk 1]
```diff
@@ -37,8 +37,8 @@ WINDOWS_SKIP_TARGETS = [
 NO_HTTP3_SKIP_TARGETS = [
     "envoy.quic.crypto_stream.server.quiche",
     "envoy.quic.deterministic_connection_id_generator",
-    "envoy.quic.crypto_stream.server.quiche",
```
Comment (danzh2010) [inline bazel/repositories.bzl:40]: Can you comment what kinda extension needs to be listed here? Why envoy.quic.crypto_stream.server.quiche doesn't?

Comment (jwendell) [reply bazel/repositories.bzl:40]: It does. I'm not removing it. Its entry is duplicated.

Comment (danzh2010) [reply bazel/repositories.bzl:40]: Woop! That's right.

**Review by danzh2010 — APPROVED** [review]

**Review by adisuissa — APPROVED** [review]
LGTM, thanks!

**Pull request merged**